src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.service;
+
+import java.util.List;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallService {
+    CinemaHall add(CinemaHall cinemaHall);
+
+    CinemaHall get(Long id);
```

Comment:
Refer to the common mistakes:
<img width="664" alt="image" src="https://github.com/mate-academy/hibernate-create-movie-session-hw/assets/140437083/65006c51-8fba-40e8-89c4-0323a0aaef90">